spec/replace.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (sentence, word1, word2){
```

Comment:
on this project the idea is to reverse engineer the function...so it defeats the purpose if you use ```replace()``` for ```replace``` 😏